index.html

```diff
@@ -23,6 +23,19 @@
         <link rel="stylesheet" href="css/daterangepicker.css">
         <link rel="stylesheet" href="css/jquery.nouislider.css">
         <link rel="stylesheet" href="css/style.css">
+        <!-- zac's formValidationScript -->
+        <script>
+function validateForm() {
+    var x = document.forms["myForm"]["email"].value;
+    var atpos = x.indexOf("@");
+    var dotpos = x.lastIndexOf(".");
+    if (atpos<1 || dotpos<atpos+2 || dotpos+2>=x.length) {

```

Comment:
try this out with the emails `"aa@@@.12"` or `a@...12321`. I'd check out a regex like [this one](https://stackoverflow.com/questions/46155/how-to-validate-email-address-in-javascript)